### DISCUSSION TITLE
Making progress on this fork / dealing with PRs

### DISCUSSION THREAD

**Discussion by smichel17** [PaperWM-community/PaperWM#19] [General]
We have a problem: we are steadily accumulating open PRs (getting close to the same number as the main repo, 9 here vs 12 there). The goal here is to facilitate working together on fixing PaperWM for GNOME 40. If PRs go for too long without being merged, then it's not so different from everyone working on their own fork, and we may start running into merge conflicts. Since I proposed making this soft fork, I feel some responsibility for fixing the situation before it gets too much worse and makes things harder to merge back upstream.

I'd like to try a governance model called\* a Temporary Benevolent Dictator (TBD). It's like a BDFL (Benevolent Dictator For Life), where one person gets to unilaterally make all decisions for the project. However, the purpose is to discover who cares about being part of the decision-making process… often as a result of someone feeling disappointed/annoyed that they didn't get a say in the decision. It's like how flipping a coin to make a decision sometimes helps you realize that you actually cared about the outcome. Once the TBD figures out who should be part of which decisions, they delegate/relinquish their authority in that area, and eventually step out of the role entirely.

*\*Although I'm sure the governance model has been used before, I believe we coined the term at Snowdrift.coop last summer, where I assumed the role of TBD for a month (July 2021) in order to help get the project moving again/faster.*

Anyway, here's the specific things I plan to do:

1. Close PRs which are unrelated to Gnome 40+ support. Perhaps later we can consider merging a bunch of this stuff to a different branch temporarily, but it gets in the way of the work of updating the existing features on gnome 40.

2. Request reviews on PRs, and merge any PRs which are not horribly broken, even if they are incomplete. The real PR, which we should care about, is https://github.com/paperwm/PaperWM/pull/396. Everything here is just collaboration towards getting that PR ready for review/merge. The main reason to wait to merge is if the changes are being actively worked on, so we don't have two people working on the same thing at once. Otherwise, merging sooner lets someone else pick up where that PR left off.

    - I'm actually quite busy right now (just started a new job), so I don't have the bandwidth to do reviews myself. @cstich @Gelbana @hikhvar @michaelhthomas @tadfisher @YannikSc (people with merge permissions) and @mskorzhinskiy @soraxas @paradajz (other people who sent PRs): **Are you up for being pinged to do a code review on an open PR?**

3. Rewrite the README notice to be clearer about items 1 and 2.

**Comment by michaelhthomas** [comment]
I'd be more than willing to be pinged and help do some PR reviews! The TBL model (™) definitely sounds like a great route to go down to make sure things keep chugging along. Thanks for being willing to step in and help out!

**Comment by soraxas** [comment]
Alrighty sounds good to me 👍🏼 I'm willing to review PRs

**Comment by YannikSc** [comment]
I'm also open for this. But I'm also having some time constrains at the moment, so I may not get to reviews
